src/libraries/Microsoft.PowerFx.Connectors/Execution/OpenApiFormUrlEncoder.cs

```diff
@@ -0,0 +1,194 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Dynamic;
+using System.Linq;
+using System.Text;
+using Microsoft.OpenApi.Models;
+
+namespace Microsoft.PowerFx.Connectors.Execution
+{
+    internal class OpenApiFormUrlEncoder
```

Comment:
>OpenApiFormUrlEncoder [](http://example.com/codeflow?start=19&length=21)

Is there an existing generic method to convert Json to FormUrl? If so, we could just use that.

Reply:
Haven't found any
Anyway, we need FormulaValue array -> URL encoded string